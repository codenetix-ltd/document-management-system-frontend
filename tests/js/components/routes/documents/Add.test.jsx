import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import DocumentAdd from 'Components/routes/documents/Add';

describe('Documents Add', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<DocumentAdd />);
  });

  it('should render correctly', () => {
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
