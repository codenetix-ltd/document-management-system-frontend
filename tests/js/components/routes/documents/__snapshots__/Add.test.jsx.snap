// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Documents Add should render correctly 1`] = `
<div>
  <ContentHeader
    breadcrumbs={
      Array [
        Object {
          "iconCls": "fa fa-copy",
          "pageLink": "/documents/list",
          "pageName": "Documents",
        },
        Object {
          "iconCls": "fa fa-plus",
          "pageLink": "/documents",
          "pageName": "Document create",
        },
      ]
    }
    title="Document create"
  />
  <ContentWrapper
    boxClass=""
    noBox={true}
  >
    <Uncontrolled(Tabs)
      id="tabs"
    >
      <Tab
        eventKey={1}
        title="Content"
      >
        <withRouter(Connect(DocumentContent)) />
      </Tab>
    </Uncontrolled(Tabs)>
  </ContentWrapper>
</div>
`;
